### ISSUE TITLE
Problem: Update package status is an undocumented administration feature of storage service locations

### ISSUE THREAD

**Issue by ross-spencer** [archivematica/Issues#746]
**Version of the documentation**

All versions. 

**Description of the issue**

There is a feature in the storage service, for protocol types that implement this, to [update_package_status](https://github.com/artefactual/archivematica-storage-service/search?q=update_package_status&unscoped_q=update_package_status). 

**Suggested fix**

It would be great to put some documentation behind this to increase its visibility (and potential) to users.

**Comment by sallain** [comment]
@ross-spencer Am I reading it right that this feature is present for Arkivum and LOCKSS spaces only? I'm not sure how a user would interact with this feature - I don't believe I've ever seen anything in the UI, so maybe it's an API thing?

**Comment by sallain** [comment]
Answered my second question - it's this feature:

![image](https://user-images.githubusercontent.com/4612276/59532624-2946e600-8e9e-11e9-8246-d9e8cc34ec4f.png)

If you try to use it on an unsupported space, you get the error shown in the screenshot above.

**Comment by ross-spencer** [comment]
Yep, @sallain sorry you had to reverse engineer that, a screenshot would have made all the difference. It does look like LOCKSS and Arkivum. And then for all other storage options, it is possible to generate that error message. So the docs can probably be pretty light when we get around to it.

**Comment by replaceafill** [comment]
This issue is being closed as part of a bulk closure process. Please click [here](https://groups.google.com/u/1/g/archivematica/c/Sq7uw1zloTA) for further information.
